sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/CLIService.java

```diff
@@ -154,6 +154,7 @@ public synchronized void start() {
       throw new ServiceException("Unable to connect to MetaStore!", e);
     }
     finally {
+      // IMetaStoreClient is not AutoCloseable, closing it manually
```

Comment:
I wouldn't bother with this. This is a copy of some Hive code anyway

Reply:
Done!